src/pages/_theme.vue

```diff
@@ -5,59 +5,159 @@ meta:
-    </div>
-    <div class="flex-1">
-      <AppColor name="bg-link" shade="Link" />
-      <AppColor name="bg-link-10" shade="10" />
-      <AppColor name="bg-link-30" shade="30" />
-      <AppColor name="bg-link-70" shade="70" />
-      <AppColor name="bg-link-110" shade="110" />
-    </div>
-    <div class="flex-1">
-      <AppColor name="bg-warning" shade="Warning" />
-      <AppColor name="bg-warning-10" shade="10" />
-      <AppColor name="bg-warning-30" shade="30" />
-      <AppColor name="bg-warning-70" shade="70" />
-    </div>
-    <div class="flex-1">
-      <AppColor name="bg-success" shade="Success" />
-      <AppColor name="bg-success-10" shade="10" />
-      <AppColor name="bg-success-30" shade="30" />
-      <AppColor name="bg-success-70" shade="70" />
-      <AppColor name="bg-success-110" shade="110" />
-    </div>
-    <div class="flex-1">
-      <AppColor name="bg-danger" shade="Danger" />
-      <AppColor name="bg-danger-10" shade="10" />
-      <AppColor name="bg-danger-30" shade="30" />
-      <AppColor name="bg-danger-70" shade="70" />
-      <AppColor name="bg-danger-110" shade="110" />
-    </div>
-    <div class="flex-1">
-      <AppColor name="bg-grey" shade="Grey" />
-      <AppColor name="bg-grey-lighter" shade="Lighter" />
-      <AppColor name="bg-grey-light" shade="Light" />
-      <AppColor name="bg-grey-dark" shade="Dark" />
-      <AppColor name="bg-grey-darker" shade="Darker" />
-    </div>
-  </div>
+  </section>
+
+  <section class="my-5">
+    <h2 class="mb-2 font-title text-2xl">Components</h2>
+
+    <section class="my-5">
+      <h3 class="mb-2 font-title text-xl">Buttons</h3>
+      <div class="bg-white p-4 flex flex-wrap">
+        <div class="m-2">
+          <AppButton variant="primary">primary</AppButton>
+        </div>
+        <div class="m-2">
+          <AppButton variant="link">link</AppButton>
+        </div>
+        <div class="m-2">
+          <AppButton variant="danger">danger</AppButton>
+        </div>
+        <div class="m-2">
+          <AppButton variant="primaryOutlined">primaryOutlined</AppButton>
+        </div>
+        <div class="m-2">
+          <AppButton variant="linkOutlined">linkOutlined</AppButton>
+        </div>
+        <div class="m-2">
+          <AppButton variant="dangerOutlined">dangerOutlined</AppButton>
+        </div>
+        <div class="m-2">
+          <AppButton variant="text">text</AppButton>
+        </div>
+        <div class="m-2">
+          <AppButton variant="dangerText">dangerText</AppButton>
+        </div>
+      </div>
+    </section>
+
+    <section class="my-5">
+      <h3 class="mb-2 font-title text-xl">AppSlider</h3>
+      <div class="bg-white p-4">
+        <div class="max-w-[1200px] mx-auto relative">
+          <AppSlider :show-navigation-buttons="true" :infinite="true">
+            <!-- Put your slides here -->
+            <template #slides>
+              <div class="flex-none w-full bg-danger">
```

Comment:
I wonder if a `AppSliderSlide` (or a better name) would be helpful here, as I guess `flex-none w-full` will be needed on all slides. It might be a bit overkill though?

```vue
<AppSlider ...>
  <AppSliderSlide></AppSliderSlide>
  <AppSliderSlide></AppSliderSlide>
  <AppSliderSlide></AppSliderSlide>
</AppSlider>

Reply:
Good idea